build/add_head.mjs

```diff
@@ -0,0 +1,35 @@
+import fs from "node:fs";
+import path from "node:path";
+
+const favicon = `<!-- Favicons -->
```

Comment:
You can drop this `<!-- Favicons -->` comment that came from Bootstrap originally and has been dropped recently